roles/prometheus/tasks/main.yml

```diff
@@ -1,53 +1,113 @@
 ---
-- name: Install prometheus
-  apt: name={{item}} state=latest update_cache=true cache_valid_time=600
+- name: Create prometheus directory structure
+  file: >
+    state=directory
+    path={{ item }}
+    owner=root
+    group=root
+  with_items:
+    - "{{ prometheus_root_dir }}"
+    - "{{ prometheus_dist_dir }}"
+    - "{{ prometheus_config_dir }}"
+    - "{{ prometheus_config_dir }}/conf.d"
```

Comment:
This isn't used. Please avoid indirection, I don't want to have to search across files to figure out what's going on.